### PR TITLE
issue: 321712 ****************** VMA 8.1.1 *******************

### PR DESCRIPTION
**Pull request by OphirMunk**
Signed-off-by: Ophir Munk <ophirmu@mellanox.com>

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-redmine-issue/74/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/176/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/191/ for details (Mellanox internal link).

**Comment by Di0gen** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-redmine-issue/75/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/192/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/177/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/193/ for details (Mellanox internal link).

**Comment by rosenbaumalex** [comment]
+1

**Comment by OphirMunk** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/183/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/199/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/189/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/205/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/213/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/226/ for details (Mellanox internal link).

File: journal.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ Added:
         - RM #820858 Update message regarding no steering rules in mlx4 to adapt to upstream drivers
         - RM #804206 DNS server (UDP,port 53) to be handled by OS by default (libvma.conf)
         - RM #786406 Jenkins - add building of source RPM 
+	- RM #823037 RHEL7.3 readiness: replace deprecated API (e.g. inet_addr) with ipv6 supporting API (e.g inet_pton)
```
Comment (rosenbaumalex) [inline journal.txt]: please remove "RHEL7.3 readiness" from message

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/214/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/227/ for details (Mellanox internal link).

**Pull request merged**